README.md

```diff
@@ -2,11 +2,26 @@
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=develop&project=mosip_pre-registration-ui&metric=alert_status)](https://sonarcloud.io/dashboard?branch=develop&id=mosip_pre-registration-ui)
 
 
-# Pre-Registration Portal
+## Overview
 
-This repository contains UI code for Pre-Registration portal. Refer [here](https://docs.mosip.io/1.2.0/modules/pre-registration/pre-registration-user-guide) for details about the functions that a resident can perform via this portal.
+MOSIP provides a reference implementation of the pre-registration portal that customized as per country needs. This repository contains UI code for Pre-Registration portal. 
 
-## Build &amp; Deployment Guide
+## Features
+The Pre-Registration Portal streamlines the identity registration process by allowing residents to submit demographic details and documents online, reducing wait times and improving data accuracy. 
```

Comment:
Features are:
1. Resident can login into PreRegistration with email id / phone number
2. Resident can create multiple PreRegistration application.
3. For each  PreRegistration application, resident can give demographic data in multiple languages.
4. For each  PreRegistration application, resident can upload various proof documents.
5. For each  PreRegistration application, resident can search for a Registration Center and search for suitable appointment slots.
6. For each  PreRegistration application, resident can book an appointment at suitable Registration Center 
7. For every appointment booking, resident can get the acknowledgement as well as a notification via email id / phone number.
8. Resident can cancel the appointment or reschedule it as well.